index.html

```
@@ -478,15 +477,33 @@ <h2 class="accordion-header bg-transparent" id="flush-headingSeven">
+          </div>
+        </div>
+
+        <div class="accordion-item bg-transparent">
+          <h2 class="accordion-header bg-transparent" id="flush-headingNine">
+            <button class="accordion-button collapsed bg-transparent" type="button" data-bs-toggle="collapse"
+              data-bs-target="#flush-collapseNine" aria-expanded="false" aria-controls="flush-collapseNine">
               <img src="pictures/windows-32x32.png" class="accordionLogo" alt="Windows" /> Windows
             </button>
           </h2>
-          <div id="flush-collapseEighth" class="accordion-collapse collapse bg-transparent" aria-labelledby="flush-headingEighth" data-bs-parent="#accordionDownload">
+          <div id="flush-headingNine" class="accordion-collapse collapse bg-transparent" aria-labelledby="flush-headingNine" data-bs-parent="#accordionDownload">
```

HTMLHint (1.9.2)

[error] The id value [ flush-headingNine ] must be unique.

(id-unique)